code/modules/admin/holder2.dm

```diff
@@ -165,12 +170,28 @@ GLOBAL_PROTECT(href_token)
 		owner.holder = null
 		owner = null
 
+/datum/admins/proc/feedback_link()
+	if(!(world.time - last_forum_access_time) > 100)
+		return cached_forum_link
+
+	last_forum_access_time = world.time
+
+	var/datum/db_query/feedback_query = SSdbcore.NewQuery("SELECT feedback FROM [format_table_name("admin")] WHERE ckey = '[key_name_admin(usr)]'")
```

Comment:
That does not look right at all (`usr`, and `key_name_admin`), this should be:

```suggestion
	var/datum/db_query/feedback_query = SSdbcore.NewQuery("SELECT feedback FROM [format_table_name("admin")] WHERE ckey = '[holder.ckey]'")
```

Do you need help learning how to set up the DB?